simbot-component-kritor-core/src/jvmMain/kotlin/love/forte/simbot/component/kritor/core/message/KritorContactElement.kt

```diff
@@ -0,0 +1,60 @@
+package love.forte.simbot.component.kritor.core.message
+
+import io.kritor.message.*
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ *
+ * Kritor 的 Contact
+ *
+ * @author Roy
+ */
+@Serializable
+@SerialName("kritor.m.contact")
+public class KritorContactElement private constructor(
+    /**
+     * 场景
+     *
+     * @see io.kritor.event.ContactElement.getScene
+     */
+    public val scene: Scene,
+    public val peer: String
+) : KritorMessageElement, KritorSendElementTransformer {
+            public companion object {
+                /**
+                 * 使用 [io.kritor.event.ContactElementOrBuilder] 构建 [KritorContactElement].
+                 */
+                @JvmStatic
+                @JvmName("valueOf")
+                public fun io.kritor.event.ContactElementOrBuilder.toKritorContact(): KritorContactElement {
+                   val scene = when(requireNotNull(scene){ "Required `scene` was null." }) {
+                       io.kritor.event.Scene.FRIEND -> Scene.FRIEND
+                       io.kritor.event.Scene.GROUP -> Scene.GROUP
+                       io.kritor.event.Scene.UNRECOGNIZED -> Scene.UNRECOGNIZED
+                       io.kritor.event.Scene.NEARBY -> Scene.NEARBY
+                       io.kritor.event.Scene.GUILD -> Scene.GUILD
+                       io.kritor.event.Scene.STRANGER -> Scene.STRANGER
+                       io.kritor.event.Scene.STRANGER_FROM_GROUP -> Scene.STRANGER_FROM_GROUP
+                   }
```

Comment:
此处大概可以简化为 `scene.resolve()`, 参考[此处](https://github.com/simple-robot/simbot-component-kritor/blob/b581272022c84e3e5088fff3232d4f30da2a1ee5/simbot-component-kritor-core/src/jvmMain/kotlin/love/forte/simbot/component/kritor/core/message/MessageElementResolver.kt#L453-L461)
```suggestion
                   val scene = scene.resolve()
```

Reply:
已修改提交